package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serlo-abc",
-  "version": "0.1.0",
+  "version": "1.1.0",
```

Comment:
Is there are specific reason why you changed the version number?

Reply:
This happened (apparently) by mistake, I should've checked git diff before committing!

Reply:
No problem, that's what code reviews are for :)